<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockRemindDialog</class>
 <widget class="QDialog" name="StockRemindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="text">
    <string>开盘后涨跌幅超过</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="text">
    <string>股价上涨达到或高于</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="text">
    <string>股价下跌达到或低于</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRise">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFall">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 11pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>150</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>(一条设置最多提示一次)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StockRemindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StockRemindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
